notebooks/dataAccess/siphon-RemoteOpen.ipynb

```diff
@@ -0,0 +1,378 @@
+   "source": [
+    "## 2. Open the dataset using `remote_open`\n",
+    "\n",
+    "We'll now use Siphon's `remote_open` to obtain a file-like object representing the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_file = ds.remote_open()\n",
+    "data_file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now have a object that we can read similar to a local file. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = data_file.readline()\n",
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Note:* When we use `remote_open` to read a dataset, we are reading raw data from a file-like object, rather than formatted data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a href=\"#top\">Top</a>\n",
+    "\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Read the returned object like a local file\n",
+    "We can now read our dataset using random access."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can read a line, as we did in the previous section, or we can read a specified number of bytes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = data_file.read(100)\n",
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can change our position in the file using `seek`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data_file.seek(0) # go to start of file\n",
+    "print(data_file.read(4)) # print first 4 bytes\n",
+    "data_file.seek(50) # go to position 50\n",
+    "print(data_file.read(10)) # print 10 more bytes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And we can read the data directly into a byte array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b = bytearray(100) # create a byte array of length 100\n",
+    "data_file.readinto(b) # read 100 bytes into the byte array\n",
+    "b[:]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Calling `getbuffer` returns the location in memory where the dataset is being stored locally."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "b = data_file.getbuffer()\n",
+    "b"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can use the memory buffer to make local writes."
```

Comment:
Again, just to clarify, by "make local writes" you mean you are actually changing the contents of `data_file`, right?